docs/Design/Exceptions/Exception Throwing.md

```diff
@@ -0,0 +1,123 @@
+# Exception Throwing
+
+A manual exception throw merely calls the declared exception class, and is treated like a normally generated exception of the same type.

```

Comment:
The word "merely" does not add to the sentence (and almost never does). No need for it.